config/application.rb

```diff
@@ -8,14 +8,10 @@
 
 module PhlexComponentsLibrary
   class Application < Rails::Application
-    config.autoload_paths << "#{root}/app/views"
-    config.autoload_paths << "#{root}/app/views/layouts"
-    config.autoload_paths << "#{root}/app/views/components"
-
-    config.autoload_paths << "#{root}/lib"
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.2
 
+    config.active_support.to_time_preserves_timezone = :zone
```

Comment:
I think we can load_defaults to 8.0 since our application is simple and I don't think new frameworks will cause us trouble.